packages/sql/package.json

```diff
@@ -29,16 +29,14 @@
     "value"
   ],
   "dependencies": {
-    "fix-esm": "^1.0.1",
-    "sql-ts": "5"
```

Comment:
@Jytesh not sure if `5` was locked for a reason, waiting to see if tests pass green under `6`

Reply:
oh ok, `fix-esm` was added to be possible to use `sql-ts@5`.